src/components/Actions/Actions.vue

```diff
@@ -360,6 +360,54 @@ export default {
 </script>
 ```
 
+### Scoped slot
+
+```vue
+<template>
+	<div>
+		<input ref="input" />
+		<Actions>
+			<template #default="{ clearFocusTrap }">
+				<ActionButton @click="focusInput(clearFocusTrap)" :close-after-click="true">
+					<template #icon>
+						<Plus :size="20" />
+					</template>
+					Focus input
+				</ActionButton>
+				<ActionButton @click="actionDelete">
+					<template #icon>
+						<Delete :size="20" />
+					</template>
+					Delete
+				</ActionButton>
+			</template>
+		</Actions>
+	</div>
+</template>
+
+<script>
+import Delete from 'vue-material-design-icons/Delete'
+import Plus from 'vue-material-design-icons/Plus'
+
+export default {
+	components: {
+		Delete,
+		Plus,
+	},
+	methods: {
+		actionDelete() {
+			alert('Delete')
+		},
+		async focusInput(clearFocusTrap) {
+			await clearFocusTrap({ returnFocus: false })
+			await this.$nextTick()
+			this.$refs.input.focus()
+		},
```

Comment:
```suggestion
		focusInput(clearFocusTrap) {
			clearFocusTrap({ returnFocus: false })
			this.$nextTick(() => this.$refs.input.focus())
		},
```
For some reasong the netlify docs fails on aysnc methods, so this is a workaroung.

Reply:
Realy?! 😱 

```suggestion
		focusInput(clearFocusTrap) {
			clearFocusTrap({ returnFocus: false })
			return this.$nextTick(() => this.$refs.input.focus())
		},
```

We must keep the promise chain

Reply:
Fine with me. Although it's just an example and has no effect for the library.